### PR TITLE
Multi Ticket functionality implemented in MasterPricer messages

### PR DESCRIPTION
**Pull request by bimusiek**
Added 2 options in MasterPricer:
- 5.56 Operation: 04.20 Fare Option - Multi-Ticket - Standard mode
- 5.57 Operation: 04.21 Fare Option - Multi-Ticket - Weighted mode

### CODE REVIEW

File: src/Amadeus/Client/Struct/Fare/BaseMasterPricerMessage.php [hunk 1]
```diff
@@ -96,12 +96,13 @@ protected function loadPassenger($passenger, &$counter, &$infantCounter)
      * @param FareMasterPricerTbSearch|FareMasterPricerCalendarOptions|TicketCheckEligibilityOptions $options
      * @return void
      */
-    protected function loadNrOfPaxAndResults($options)
+    protected function loadNrOfPaxAndResultsAndMultiTicketWeights($options)
```
Comment (DerMika) [inline src/Amadeus/Client/Struct/Fare/BaseMasterPricerMessage.php]: Oh god, these names are getting out of hand! :)

**Comment by DerMika** [comment]
You can also add an entry in the CHANGELOG if you want. Or I can do it too, it's the same to me.

File: src/Amadeus/Client/Struct/Fare/BaseMasterPricerMessage.php (hunk 1 above)
Comment (bimusiek) [reply src/Amadeus/Client/Struct/Fare/BaseMasterPricerMessage.php]: I can change to `loadNumberOfUnits` 😄

Comment (DerMika) [reply src/Amadeus/Client/Struct/Fare/BaseMasterPricerMessage.php]: Nah, it's fine! Just when another option is added, we may have to reconsider the naming approach

**Comment by bimusiek** [comment]
I changed the name of the function, added changelog and 1 small check for number of units (as I forgot before and realised just now)

**Comment by DerMika** [comment]
Thanks again for another contribution!

**Comment by DerMika** [comment]
I've released version 1.5.0, with all your changes! Thanks again!

**Pull request merged**